docs/coins_spends_and_wallets.md

```diff
@@ -265,8 +265,7 @@ This is because all wallets already know and agree on what the standard format f
 They know what their own pubkeys are, so when a new coin is created a wallet can check if the puzzle inside that coin is a 'standard send puzzle' to one of their pubkeys.
 If it is, then that coin can be considered to be owned by that 'wallet' as nobody else can spend it.
 
-If the wallet that 'owns' the coin then wanted to send that coin on again to somebody else, they would generate a 'standard send puzzle' but with the new recipient's pubkey.
-They could then spend the coin that they own, destroying it, and creating a new coin that is locked up with the new recipients pubkey in the process.
+If the wallet that 'owns' the coin then wanted to send that coin on again to somebody else, they ask for an address (which is just a bech32m encoded puzzle hash) and then they could then spend the coin that they own, destroying it, and creating a new coin that is locked up with the new recipients puzzle hash in the process.
```

Comment:
Consider being a bit more concise by removing extra words

~~just~~

~~in the process~~